nodejs/eks/package.json

```diff
@@ -11,9 +11,9 @@
     "homepage": "https://pulumi.io",
     "repository": "https://github.com/pulumi/pulumi-eks",
     "dependencies": {
-        "@pulumi/aws": "^1.0.0-beta",
-        "@pulumi/kubernetes": "^1.0.0-beta",
-        "@pulumi/pulumi": "^1.0.0-beta",
+        "@pulumi/aws": "^1.5.0",
```

Comment:
```suggestion
        "@pulumi/aws": "^1.0.0",
```

Reply:
This fix requires v1.5.0...Since that reflects terraform aws provider v2.31.0 which has the required changes for enabling cluster tagging. 
https://github.com/pulumi/pulumi-aws/blob/v1.5.0/CHANGELOG.md#150-2019-10-03